src/routes/_authed/campaigns/$dmUsername.$campaignSlug/notes/-components/editor/file-sidebar/sidebar-system-folder/location-system-folder/location-category-folder.tsx

```
@@ -0,0 +1,16 @@
+import { CategoryFolderButton } from "../generic-category-folder/category-folder-button";
+import { LocationNoteContextMenu } from "./location-note-context-menu";
+import { LocationCategoryFolderContextMenu } from "./location-category-context-menu";
+import { LOCATION_CONFIG } from "~/components/forms/category-tag-dialogs/location-tag-dialog/types";
+
+interface LocationCategoryFolderProps {}
+
+export const LocationCategoryFolder = ({}: LocationCategoryFolderProps) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)